packages/at_secondary_server/lib/src/enroll/enroll_datastore_value.dart

```diff
@@ -42,4 +50,6 @@ class EnrollApproval {
   }
 }
 
+enum EnrollStatus { pending, approved, denied, revoked, expired }
```

Comment:
I believe we have moved the EnrollStatus to at_commons. Please update the at_commons version and remove it here.

Reply:
[fbdafe4](https://github.com/atsign-foundation/at_server/pull/1536/commits/fbdafe4e55ca0a130f7e004f143d7f74d7437942)